lib/exercises.rb

```diff
@@ -1,19 +1,52 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2) or O(m * n) where m is the number of words and n is the characters
+# Space Complexity: O(n) where n is the number of words in the array
 
 def grouped_anagrams(strings)
```

Comment:
You are right that it's O(nm) which isn't O(n^2) unless n and m are the same.  On the other hand if the words are all relatively small (like English words, you can call this O(n).